package.json

```diff
@@ -3,33 +3,31 @@
   "version": "1.5.0",
   "private": true,
   "license": "GPL-3.0-only",
-
   "scripts": {
-    "build": "vue-cli-service build",
-    "build:report": "vue-cli-service build --report",
-    "build:watch": "vue-cli-service build --watch --mode development",
+    "build": "vite build",
+    "build:watch": "yarn run build --watch --mode development",
     "checkpoint": "./src/services/mock/generate",
-    "coverage:unit": "nyc vue-cli-service test:unit",
-    "coverage:e2e": "bash -c 'concurrently --names \"SERVER,TESTS\" --prefix-colors \"yellow,blue\" --success \"first\" --kill-others \"yarn run serve:jupyterhub\" \"nyc vue-cli-service test:e2e --mode offline ${0}\"'",
+    "coverage:unit": "vitest run --coverage",
+    "coverage:e2e": "COVERAGE=true yarn run test:e2e",
     "dev": "yarn run serve",
-    "docs": "jsdoc -c jsdoc.conf.json",
-    "lint": "vue-cli-service lint",
-    "serve": "concurrently --names 'SERVER,CLIENT' --prefix-colors 'yellow,blue' --success \"first\" --kill-others 'yarn run serve:jupyterhub' 'yarn run serve:vue'",
-    "serve:vue": "vue-cli-service serve --mode offline",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx,.vue --ignore-path .gitignore",
+    "preview": "node ./scripts/concurrently.js serve:jupyterhub preview",
+    "serve": "node ./scripts/concurrently.js serve:jupyterhub serve:vue",
+    "serve:vue": "vite --mode offline",
     "serve:jupyterhub": "nodemon src/services/mock/json-server.js",
-    "test:e2e": "bash -c 'concurrently --names \"SERVER,TESTS\" --prefix-colors \"yellow,blue\" --success \"first\" --kill-others \"yarn run serve:jupyterhub\" \"vue-cli-service test:e2e --mode offline ${0}\"'",
-    "test:unit": "vue-cli-service test:unit",
-    "test": "yarn run test:unit || exit 1; yarn run coverage:e2e --headless"
+    "test:component": "cypress open --component",
+    "test:e2e": "yarn run serve e2e:open",
+    "test:unit": "vitest",
+    "test": "vitest run && yarn run serve cy:run"
```

Comment:
Could we make the concurrently script do this too?

(In my head this command runs the e2e headless because it's a generic check that everything passes)

Reply:
This is `&&` (do this, then do that) not `&` (do this in the background and do that)